pkg/storaged/vdo-details.jsx

```diff
@@ -266,8 +265,7 @@ export class VDODetails extends React.Component {
                         <StorageButton kind="danger" onClick={delete_}>{_("Delete")}</StorageButton>
                     </>,
                 }}>
-                    <CardTitle><Text component={TextVariants.h2}>{cockpit.format(_("VDO device $0"), vdo.name)}</Text></CardTitle>
-
+                    <CardTitle component="h2">{cockpit.format(_("VDO device $0"), vdo.name)}</CardTitle>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18746-20230502-181707-d585bac8-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)